packages/playground/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
   production: true,
+  apiURL: 'https://ng-query-backend.vercel.app'
```

Comment:
@NetanelBasal That's a valid concern, I didn't think of it previously. I tried opening two browsers at the same time and clicking the button, they will see the unexpected result. 
What is your suggestion to fix that?

Reply:
I'm not familiar with vercel. Is there any way to reset the server with a command?

Reply:
@NetanelBasal Sadly I am not familiar with Vercel too. Choose Vercel to deploy this express app because it seems to be the easiest 😂

As my understanding, it is not really a server. They called this feature `Serverless Functions`

https://vercel.com/docs/concepts/functions/serverless-functions

What about for each browser visit, we generate a unique id and send it together to the API. Where on the server, instead of keep a single instance of `todos` array, we keep dictionary of `{ [key: string]: Todo }`

So that the data is unique for each session?

Reply:
Yes, we can do it for now.

Reply:
@NetanelBasal Noted, I'll work on it. Should I push the changes to this same PR? I think yes because after we merge the PR, then everything is there.

Reply:
sure

Reply:
@NetanelBasal I pushed the changes, not sure if it is good enough. 
I also have to do `exhaustMap` on the playground code to not fire multiple events to the backend. The reason is we don't have any locking mechanism therefore data would be overridden when you click the button multiple times. 
I think there is still a bug when you click the `Add todo` and `Add todo v2` at the same time could override the data, but haven't thought of a better way to fix it 😂

Could you please try it on the playground and let me know your thought?
https://trungk18.github.io/ng-query/

https://user-images.githubusercontent.com/6767322/196841715-b6b5bb36-7d1b-485a-bec4-49e9247dd1ca.mp4

Reply:
It makes everything complicated. What about get rid of the server and use this service:

https://crudcrud.com/